### DISCUSSION TITLE
Feature request: support the sops asdf plugin

### DISCUSSION THREAD

**Discussion by jessesimpson36** [renovatebot/renovate#21304] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

I'd like to add the sops asdf plugin so that my downstream renovate configurations don't have to specifically look for it. I see that in /lib/modules/manager/asdf/upgradeable-tooling.ts, there is a list of plugins for this purpose, but sops is not yet included.

I already have a [fork](https://github.com/jessesimpson36/renovate/commit/afce7be23a5b8d01497a0063b394527775528957) with this change if you'd like to take a look. However, the contribution guidelines state that I must open a discussion first before opening a PR.

**Comment by viceice** [comment]
please open a PR for this trivial change ❤️

**Reply by jessesimpson36** [reply]
You got it. One moment.

**Reply by jessesimpson36** [reply]
https://github.com/renovatebot/renovate/pull/21306
